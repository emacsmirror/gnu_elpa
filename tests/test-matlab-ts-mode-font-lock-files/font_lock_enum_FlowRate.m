% -*- matlab-ts -*-
% See https://github.com/acristoffers/tree-sitter-matlab/issues/28
classdef font_lock_enum_FlowRate < int32
   enumeration
      Low    (10)
      Medium (FlowRate.Low*5)
      High   (FlowRate.Low*10)
   end
end
