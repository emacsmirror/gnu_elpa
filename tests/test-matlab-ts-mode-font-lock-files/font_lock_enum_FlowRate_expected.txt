c ccc ccccccccc ccc
c ccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
kkkkkkkk fffffffffffffffffffffff o fffff
   kkkkkkkkkkk
      PPP    bnnb
      PPPPPP bddddddddDdddonb
      PPPP   bddddddddDdddonnb
   kkk
kkk
